heron/scheduler-core/src/java/com/twitter/heron/scheduler/SubmitterMain.java

```diff
@@ -189,7 +189,7 @@ private static Options constructOptions() {
         .build();
 
     Option topologyJar = Option.builder("j")
-        .desc("user heron topology jar/pex file path")
+        .desc("user heron topology jar/tar/pex file")
         .longOpt("topology_bin")
         .hasArgs()
         .argName("topology binary file")
```

Comment:
Is there a way to covey that this doesn't actually represent a file or path on the local disk, but the name of the file that will be invoked on the executor?